code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -99,6 +142,42 @@
 				H.legcuffed = null
 	qdel(src)
 
+
+/obj/item/weapon/grenade/iedcasing/proc/process_shrapnel()
+
+	if(shrapnel_list.len > 0)
+		var/atom/target
+		var/atom/curloc = get_turf(src)
+		var/list/possible_targets= trange(7, curloc)
+		var/list/bodyparts = list("head","chest","groin","l_arm","r_arm","l_hand","r_hand","l_leg","r_leg","l_foot","r_foot")
+		for(var/obj/item/shrapnel in shrapnel_list)
+			if(istype(shrapnel, /obj/item/ammo_casing))// If the shrapnel is a bullet casing it will be fired
+				var/obj/item/ammo_casing/shrapnel_bullet = shrapnel //shitcode but otherwise BB is undefined var
+				var/obj/item/projectile/shrapnel_projectile = null
+				target=pick(possible_targets)
+				if(shrapnel_bullet.BB)
+					shrapnel_projectile = shrapnel_bullet.BB
+				else
+					shrapnel_projectile = new /obj/item/projectile/bullet/shrapnel/small(src)
+				shrapnel_projectile.forceMove(curloc)
+				shrapnel_projectile.launch_at(target,bodyparts[rand(1,bodyparts.len)],curloc,src)
+				qdel(shrapnel) // the casing is disintegrated in the explosion
+			else // elif doesnt work even though its in the dm ref
```

Comment:
`else if` does work, `elif` is used for preprocessor conditionals, using `ifdef`.